pkg/utils/kubelessutil.go

```diff
@@ -641,6 +641,8 @@ func EnsureFuncDeployment(client kubernetes.Interface, funcObj *kubelessApi.Func
 				Value: dpm.Spec.Template.Spec.Containers[0].Resources.Limits.Memory().String(),
 			},
 		)
+	} else {
+		return fmt.Errorf("Expected non-empty handler and non-empty function content")
```

Comment:
We should not return an error here because there is the case in which the user specifies an already built image (custom) that doesn't require a handler and function. For example: `kubeless function deploy foo --runtime-image=nginx` is a valid command. I am okay if we print something here in the logs though because probably something weird is happening.